### PR TITLE
Aqa/js conditions loops

### PR DESCRIPTION
**Pull request by Denys-Naumenko**
This merge request serves as a practical exercise for reinforcing understanding of conditions and loops in JavaScript.

### CODE REVIEW

File: js-conditions-loops/conditionsAndloops.js [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+
+if (numberA > numberB) {
+  biggerNumber = "The bigger number is: " + numberA;
+} else if (numberA < numberB) {
+  biggerNumber = "The bigger number is: " + numberB;
+} else {
+  biggerNumber = "The numbers are equal.";
+}
+
+console.log(chalk.greenBright(biggerNumber));
```
Comment (landriiuk) [inline js-conditions-loops/conditionsAndloops.js]: you have to write it using the ternary operator. https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Conditional_operator

File: js-conditions-loops/conditionsAndloops.js [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+  if (currentChar === "a") {
+    index++;
+    continue;
+  }
+
+  if (currentChar === "n") {
+    break;
+  }
+
+  formatedMessage += currentChar;
```
Comment (landriiuk) [inline js-conditions-loops/conditionsAndloops.js]:
```suggestion
  formattedMessage += currentChar;
```

File: js-conditions-loops/conditionsAndloops.js [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+  biggerNumber = "The bigger number is: " + numberB;
+} else {
+  biggerNumber = "The numbers are equal.";
+}
+
+console.log(chalk.greenBright(biggerNumber));
+
+
+let rawMessage = "Javascript is fun";
+let formatedMessage ="";
```
Comment (landriiuk) [inline js-conditions-loops/conditionsAndloops.js]:
```suggestion
let formattedMessage ="";
```

File: js-conditions-loops/conditionsAndloops.js [hunk 4]
```diff
@@ -0,0 +1,117 @@
...
+    employeeSalary = 333;
+    employeeBonus = employeeSalary * 0.03;
+  } else if (employeeTotalExperienceInMonths < 48) {
+    employeeSalary = 555;
+    employeeBonus = employeeSalary * 0.04;
+  } else {
+    employeeSalary = 666;
+    employeeBonus = employeeTotalExperienceInMonths/12/100 * employeeSalary  ;
+  }
+}
```
Comment (landriiuk) [inline js-conditions-loops/conditionsAndloops.js]: This code works, but it's hard to read. you can simplify. 
```suggestion
let employeeExperience = 0.5;// or full month format 12 month = 1 year
let employeeSalary;
let employeeBonus;


if (employeeExperience < 0.5) { // if you're gonna use full month format then here should be 6 
    employeeSalary = 0;
    employeeBonus = 0;
} else if (employeeExperience >= 0.5 && employeeExperience < 1) {// if you're gonna use full month format then here should be 6 and 12
    employeeSalary = 100;
    employeeBonus = employeeSalary * 0.01;
}
..... 
```

**Review by landriiuk — CHANGES_REQUESTED** [review]

**Comment by landriiuk** [comment]
there few points that need to be reworked.

File: js-conditions-loops/conditionsAndloops.js (hunk 4 above)
Comment (Denys-Naumenko) [reply js-conditions-loops/conditionsAndloops.js]: fix this, please check new commit

File: js-conditions-loops/conditionsAndloops.js (hunk 1 above)
Comment (Denys-Naumenko) [reply js-conditions-loops/conditionsAndloops.js]: fix this, please check new commit

File: js-conditions-loops/conditionsAndloops.js (hunk 3 above)
Comment (Denys-Naumenko) [reply js-conditions-loops/conditionsAndloops.js]: fix this, please check new commit

File: js-conditions-loops/conditionsAndloops.js (hunk 2 above)
Comment (Denys-Naumenko) [reply js-conditions-loops/conditionsAndloops.js]: fix this, please check new commit

**Review by landriiuk — APPROVED** [review]

**Comment by landriiuk** [comment]
Well done. Thanks. You can merge this pr into main

**Pull request merged**